2019.10/setting-build-time-variables/README.txt

```diff
@@ -0,0 +1,19 @@
+
+What:
+Set variables from the "outside" when compiling (go buidl, install, run etc..)
+
+Why:
+Some information is too dynamic to hardcode in the source code, but, at the same time, not something we can figure out at runtime.
+
+The version of a program, for example, is a pain to have in in the source code beacuse it's very easy to forget to update it every time you compile a new version.
```

Comment:
There's a typo: "beacuse" instead of "because" :)

Reply:
thanks :+1: